deltacat/compute/compactor/steps/hash_bucket.py

```diff
@@ -99,6 +100,7 @@ def _group_file_records_by_pk_hash_bucket(
         sort_key_names,
         read_kwargs_provider,
         deltacat_storage,
+        **kwargs,
```

Comment:
use `deltacat_storage_kwargs` as the `hash_bucket` step itself may require kwargs.